settings.html

```diff
@@ -73,6 +74,26 @@ <h1>Settings</h1>
 <p style="font-size: small"><i>* By creating a share URL, you agree that your progress, progress URL, IP address, and time uploaded may be stored on a server.</i></p>
 </div>
 
+<div class="section">
+<div class="sectionTitle">Browser Source</div>
```

Comment:
maybe "mini tracker"? calling this feature "browser source" is like calling the checklist "html page".

"Generate a condensed tracking page that can be used in stream overlays."

Reply:
I called it a Browser Source because that's what it is called in OBS and figured a user looking for a stream overlay would be looking specifically for that term, but I see what you mean. It looks like similar features in things like StreamLabs are called Widgets. Is calling it a Widget fine? Mini tracker seems not accurate enough because this popout isn't really intended to be used outside of broadcast software.

Reply:
"(stream) widget" works great.